src/rust/runtime/network/mod.rs

```diff
@@ -22,6 +29,19 @@ pub mod types;
 // Traits
 //==============================================================================
 
+///
+/// **Brief**
+///
+/// Since IPv6 is not supported, this method simply unwraps a SocketAddr into a SocketAddrV4
+/// or returns an error indicating the address family is not supported.
+///
+pub fn unwrap_socketaddr(addr: SocketAddr) -> Result<SocketAddrV4, Fail> {
+    match addr {
+        SocketAddr::V4(addr) => Ok(addr),
+        _ => Err(Fail::new(libc::EINVAL, "bad address family"))
+    }
+}
+
```

Comment:
Would you file an issue for IPv6 support and link it here with a `FIXME` comment?